src/main/java/com/github/theholywaffle/teamspeak3/SocketReader.java

```diff
@@ -116,7 +119,7 @@ public void run() {
 
 					c.feedError(line.substring("error ".length()));
 					if (c.getError().getId() != 0) {
-						TS3Query.log.severe("TS3 command error: " + c.getError());
+						log.debug("TS3 command error: {}", c.getError());
```

Comment:
I understand your point, but shouldn't it be clear that a user should check method return values, or alternatively just turn on DEBUG logging while actually debugging his application? I'm unsure if logic should be sacrificed because of inexperienced developers.

And what about API calls that are not checked in advance for validity, through user input? I wouldn't want these command errors to be logged, I rather check if the command returns true / not null / > -1 and return a error message to the client if the input was invalid. Sure, I could configure my logging framework to just ignore logs from SocketReader, in that case it should be on warn level, as I would want to still get error logging information, and a command error wouldn't cause the API to no longer run as expected in most cases (it could of course cause follow up issues if application wasn't written in a robust way). I could also do additional API calls to prevent command errors, however that would unnecessarily decrease the performance.